entgql/template/pagination.tmpl

```diff
@@ -420,10 +429,16 @@ func (p *{{ $pager }}) orderExpr(query *{{ $node.QueryName }}) sql.Querier {
 		return sql.ExprFunc(func(b *sql.Builder) {
 			for _, o := range p.order {
 				direction := o.Direction
+				nullsDirection := o.NullsDirection
```

Comment:
There's a difference here between `direction` and `nullsDirection`. This makes me think we're doing something differently between the two that we probably shouldn't be. Since there's no empty check on `direction`, either there should be, or it's been done earlier in the callstack somewhre. If it's done earlier then we should do this in the same place.